cms/envs/test.py

```diff
@@ -129,9 +130,11 @@
     },
 }
 
-# This hack disables migrations during tests. We want to create tables directly from the models for speed.
-# See https://groups.google.com/d/msg/django-developers/PWPj3etj3-U/kCl6pMsQYYoJ.
-MIGRATION_MODULES = {app: "app.migrations_not_used_in_tests" for app in INSTALLED_APPS}
+
+# Disables migrations and creates tables directly from models for speed.
+os.environ['ENABLE_MIGRATIONS'] = '1'
```

Comment:
@andy-armstrong Sorry, I should have pointed out that this will be removed. I only added it in a separate commit because I wanted to test the migrations on Jenkins. Once this PR is merged, we will update the Jenkins config to set this variable instead.